devtools/projects/shell-chrome/src/BUILD.bazel

```
@@ -283,33 +283,40 @@ esbuild(
 
 exports_files(["index.html"])
 
-filegroup(
-    name = "prod_app_static_files",
+[filegroup(
+    name = "prod_app_static_files_" + build_target,
```

You could potentially use a more specific variable name. It seems like the value currently does not refer to a build target directly (in the context of Bazel), and you could store `[chrome, firefox]` as variable to avoid duplication

```suggestion
    name = "prod_app_static_files_" + browser,
```